pkgs/applications/audio/alsa-scarlett-gui/default.nix

```diff
@@ -24,6 +24,11 @@ stdenv.mkDerivation rec {
   sourceRoot = "source/src";
   nativeBuildInputs = [ pkg-config wrapGAppsHook4 ];
   buildInputs = [ gtk4 alsa-lib ];
+  postInstall = ''
+      substituteInPlace $out/share/applications/vu.b4.${pname}.desktop \
+        --replace "Exec=/bin/alsa-scarlett-gui" "Exec=$out/bin/${pname}" \
+        --replace "Icon=alsa-scarlett-gui" "Icon=${pname}"
```

Comment:
This replaces "alsa-scarlett-gui" with "alsa-scarlett-gui"?

Reply:
Yeah sorry those are remnants from testing that line will be removed. As its not needed.